pkg/app/pagewriter/sign_in_page.go

```diff
@@ -104,6 +105,11 @@ func loadCustomLogo(logoPath string) (string, error) {
 		return "", nil
 	}
 
+	if strings.HasPrefix(logoPath, "https://") || strings.HasPrefix(logoPath, "http://") {
```

Comment:
@JoelSpeed -- 2 security concerns:

- We shouldn't have `http://` be allowed. If the oauth2-proxy is hosted with HTTPS (which it very well should be!), HTTP would result in mixed content warnings/issues with the HTTPS hosting page.
- (Non Blocking) There's a very small security issue potentially on where this IMG is hosted and the `cookie-domain` -- if non-exact domain style settings are used (e.g. `.yourcompany.com`) the session cookies will leak to this IMG path if they are hosted at the same top level domain (potentially an issue since a lot of people use different service providers under different subdomains that have different security controls/ownership).